src/app/story/storyrow/totaltimes/totaltimes.component.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { fakeAsync, ComponentFixture, TestBed } from '@angular/core/testing';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { NO_ERRORS_SCHEMA } from '@angular/core';
+import { By } from '@angular/platform-browser';
+import { BsDropdownModule } from 'ngx-bootstrap/dropdown';
+
+import { TotaltimesComponent } from './totaltimes.component';
+import { TimeDisplayPipe } from '../../../pipes/timedisplay';
+
+
+describe('RecentsComponent', () => {
+  let fixture: ComponentFixture<TotaltimesComponent>;
+  let component: TotaltimesComponent;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        TotaltimesComponent,
+        TimeDisplayPipe,
+      ],
+      providers: [],
+      imports: [
+        HttpClientTestingModule,
+        NoopAnimationsModule,
+        BsDropdownModule.forRoot()
+      ],
+      // This is used to avoid having to import components we don't test here
+      schemas: [NO_ERRORS_SCHEMA]
```

Comment:
It's a pretty small component with few dependencies, is `NO_ERRORS_SCHEMA` necessary?